train.py

```diff
@@ -76,12 +253,12 @@ def main(args):
     # check the number of events higher than 300 GeV
     mask2 = (Yr_pt[:,0] > 300)
     Yr_pt = Yr_pt[mask2]
-    print("# of events higher than 300 GeV : {}".format(Yr_pt.shape[0]))
+    #print("# of events higher than 300 GeV : {}".format(Yr_pt.shape[0]))
 
     indices = np.array([i for i in range(len(Yr))])
-    print(indices)
-    indices_train, indices_test = train_test_split(indices, test_size=0.2, random_state= 7)
-    indices_train, indices_valid = train_test_split(indices_train, test_size=0.2, random_state=7)
+    #print(indices)
+    indices_train, indices_valid = train_test_split(indices, test_size=0.2, random_state= 7)
+    indices_train, indices_test = train_test_split(indices_train, test_size=0.2, random_state=7)
```

Comment:
i'm confused... should it be reversed?

usually, we split the data into training+validation and testing (separate)

in the end it probably doesn't matter much